src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1082,6 +1085,7 @@ namespace winrt::TerminalApp::implementation
         _rootPane->Maximize(_zoomedPane);
         // Update the tab header to show the magnifying glass
         _headerControl.IsPaneZoomed(true);
+        IsPaneZoomed(true);

```

Comment:
Idea: should we just define the setter for `IsPaneZoomed` to update the `_headerControl`? (and do the same for all these other ones).

Reply:
It is possible.. though actually we need to do something else:
*  TabHeaderControl should expose DependencyProperties rather simple properties
* We need to bind these DependencyProperties to the tab properties (manually as no XAML is available).

Probably we should create a follow up task instead. WDYT?

Reply:
_That_ sounds way better. I like that. Yeah, file a follow up task please.

Reply:
The follow up task is here: https://github.com/microsoft/terminal/issues/9043